autopush/main.py

```diff
@@ -105,7 +106,10 @@ def run(self):
         """Start the services and run the reactor"""
         reactor.suggestThreadPoolSize(constants.THREAD_POOL_SIZE)
         self.startService()
-        reactor.run()
+        try:
+            reactor.run()
+        except ConnectionDone:
```

Comment:
this one's not right but I can see what lead you towards it: our stack trace order in sentry is screwed up (logged #1134). Look at it again in reverse =]

Reply:
I'm not so sure. I was looking at https://sentry.prod.mozaws.net/operations/autopush-prod/issues/643290/. That seems to indicate that the `ConnectionDone` error is happening as part of `reactor.run()` here.

Reply:
that one's due to web.base _response_err logging it, you can see it in the next to last stack frame

Reply:
I have no idea how the hell this got reverted. Removing this try since the ConnectionDone is handled at a lower layer.